diff.txt

```diff
@@ -0,0 +1,879 @@
+  ADDED      CLAUSE     com.liferay.portal.kernel.poller
+  ADDED      CLAUSE     com.liferay.portal.kernel.portlet.bridges.mvc
+  ADDED      CLAUSE     javax.portlet
+ ADDED      HEADER     Private-Package:com.liferay.blade.samples.pollprocessor,content
+ REMOVED    HEADER     Private-Package:content
+ REMOVED    HEADER     Provide-Capability:liferay.resource.bundle;bundle.symbolic.name="blade.pollprocessor";resource.bundle.base.name="content.Language"
+ ADDED      HEADER     Provide-Capability:osgi.service;objectClass:List<String>="com.liferay.portal.kernel.poller.PollerProcessor",osgi.service;objectClass:List<String>="javax.portlet.Portlet",liferay.resource.bundle;bundle.symbolic.name="blade.pollprocessor";resource.bundle.base.name="content.Language"
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))"
+ ADDED      HEADER     Service-Component:OSGI-INF/com.liferay.blade.samples.pollprocessor.BladePollProcessor.xml,OSGI-INF/com.liferay.blade.samples.pollprocessor.BladePollProcessorPortlet.xml
+CHANGED    RESOURCES  <resources>
+ ADDED      RESOURCE   OSGI-INF/com.liferay.blade.samples.pollprocessor.BladePollProcessor.xml
+ ADDED      RESOURCE   OSGI-INF/com.liferay.blade.samples.pollprocessor.BladePollProcessorPortlet.xml
+ ADDED      RESOURCE   OSGI-OPT/src/com/liferay/blade/samples/pollprocessor/BladePollProcessor.java
+ ADDED      RESOURCE   OSGI-OPT/src/com/liferay/blade/samples/pollprocessor/BladePollProcessorPortlet.java
+ ADDED      RESOURCE   com/liferay/blade/samples/pollprocessor/BladePollProcessor.class
+ ADDED      RESOURCE   com/liferay/blade/samples/pollprocessor/BladePollProcessorPortlet.class
+
+
+blade.portlet.actioncommand-1.0.0.jar
```

Comment:
Hey lawrence.  if we don't see any diffs at all, can we remove this from the printout?  so eventually if we get 100% the same the diff will just be empty :)